pallas-network/src/miniprotocols/localstate/queries_v16/mod.rs

```diff
@@ -77,7 +77,75 @@ pub type Credential = StakeAddr;
 
 /// Updates to the protocol params as [in the Haskell sources](https://github.com/IntersectMBO/cardano-ledger/blob/d30a7ae828e802e98277c82e278e570955afc273/libs/cardano-ledger-core/src/Cardano/Ledger/Core/PParams.hs#L151)
 /// (via [`EraPParams`](https://github.com/IntersectMBO/cardano-ledger/blob/d30a7ae828e802e98277c82e278e570955afc273/libs/cardano-ledger-core/src/Cardano/Ledger/Core/PParams.hs#L255-L258)).
-pub type PParamsUpdate = ProtocolParam;
+/// Conway era protocol parameters, corresponding to [`ConwayPParams`](https://github.com/IntersectMBO/cardano-ledger/blob/d30a7ae828e802e98277c82e278e570955afc273/eras/conway/impl/src/Cardano/Ledger/Conway/PParams.hs#L512-L579)
+/// in the Haskell sources.
+/// @todo: Encoding should be handled manually, Encode derive won't be correct.
```

Comment:
Actually, the encoding **is** derived, so perhaps this comment is outdated.

Reply:
I interpreted this comment as saying that the encoding _is_ derived, and is incorrect because of it.

It's possible that it's outdated because now we have separate `PParamsUpdate` and `ProtocolParam` types. No existing queries take either of these, so I couldn't tell.

The PR #609 which introduced all these encoding bugs also commented out some network tests, and claim those tests failed because of the bug. I'll see if I can get those tests passing, and remove the todo if so.